src/Display.jl

```diff
@@ -29,11 +29,13 @@ function git_file_stream(repo::LibGit2.GitRepo, spec::String; fakeit::Bool=false
     return iob
 end
 
-function status(ctx::Context, mode::PackageMode, use_as_api=false)
+function status(ctx::Context, args::Vector{PackageSpec}, mode::PackageMode; use_as_api=false)
     env = ctx.env
     project₀ = project₁ = env.project
     manifest₀ = manifest₁ = env.manifest
     diff = nothing
+    pkgs = getproperty.(args, :name)
```

Comment:
If the uuid is given in the packagespec we should filter by that, otherwise name.